features/admin.applications.v1/models/endpoints.ts

```diff
@@ -29,6 +29,10 @@ export interface ApplicationsResourceEndpointsInterface {
      * Below route is to fetch the dcr configuration from server configurations api.
      */
     dcrConfiguration: string;
+    /**
+     * Below route is to fetch the group data from the application api.
```

Comment:
Shall we highlight the fact that group list is retrieved as `metadata` in the doc comment too?

eg:

```suggestion
     * Below route is to fetch the group list as metadata from the application API.
```

Reply:
addressed by https://github.com/wso2/identity-apps/pull/7399/commits/dc902a134667e113b931b40d444624c884e6ddbb